src/server/services/metadata.ts

```diff
@@ -495,6 +495,8 @@ export const fcdoFuneralDirectorsByCountry = {
   Estonia: "https://www.gov.uk/government/publications/estonia-list-of-funeral-directors",
   Finland: "https://www.gov.uk/government/publications/funeral-directors-in-finland",
   Gabon: "https://www.gov.uk/government/publications/gabon-list-of-funeral-directors",
+  Gambia:
```

Comment:
Should be "The Gambia" now

Reply:
I don't know why that wasn't picked up by my find and replace 🤔